src/components/login/index.js

```diff
@@ -47,7 +47,31 @@ class Login extends Component {
 		onLogin(new FormData(e.target))
 	}
 
-	render () {
+	displayLoginFields() {
```

Comment:
While this is a cool improvement, I think we should do this in other PR/issues. This has nothing to do with dealing with toot links. It's less convenient but it's cleaner.